enonic/src/main/resources/template/main.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Webpack Starter</title>
+    <script data-th-src="${portal.assetUrl({'_path=js/bundle.js'})}" src="../assets/js/bundle.js" type="text/javascript"></script>
+    <script data-th-src="${portal.assetUrl({'_path=ts/bundle.js'})}" src="../assets/ts/bundle.js" type="text/javascript"></script>
+
+    <link rel="stylesheet" data-th-href="${portal.assetUrl({'_path=styles/bundle.css'})}"  href="../assets/styles/bundle.css"/>
+
+</head>
+<body>
+    Hello from Comments!
+</body>
+</html>
```

Comment:
This is also a file automatically created by the enonic starter and we are not going to use it, if we are going to use this template we need to move the bundle script including at the end of the body. I suggest to remove it since we are going to write our own pages and parts in enonic.